bot-components/GitHub_queries.ml

```diff
@@ -443,3 +457,37 @@ let get_cards_in_column column_id =
                let pr_number = Str.matched_group 1 content_url in
                Some (pr_number, card_id)
              else None))
+
+let get_installations =
+  generic_get "installation/repositories" ~header_list:app_api_preview_header
+    (fun json ->
+      let open Yojson.Basic.Util in
+      json |> member "repositories" |> to_list
+      |> List.map ~f:(fun json ->
+             ( json |> member "owner" |> member "login" |> to_string
+             , json |> member "name" |> to_string )))
```

Comment:
What is this doing in this PR? Has this escaped from the previous version of #100?

Reply:
I messed up while rebasing.